presto-spi/src/main/java/com/facebook/presto/spi/type/SqlTime.java

```diff
@@ -19,30 +19,47 @@
 import java.time.ZoneId;
 import java.time.format.DateTimeFormatter;
 import java.util.Objects;
+import java.util.Optional;
```

Comment:
cmt msg ("Introduce new TIME/TIMESTAMP semantic to SqlTypes")
* semantic*s*
* `SqlTypes` or `SqlTime` or "SQL types" ?